.azure-ci/docker_scripts.sh

```diff
@@ -22,5 +22,10 @@ flake8 --exit-zero /io/
 pip uninstall -y pyflagser
 
 # Build wheels
-pip install wheel
-python setup.py sdist bdist_wheel
+pip install wheel==0.34.1 auditwheel==3.1.0
+python setup.py bdist_wheel
+
+# Repair wheels with auditwheel
+auditwheel repair dist/*whl -w dist/
+# remove wheels that are not manylinux2010
+rm -rf dist/*-linux*.whl
```

Comment:
Does this mean we won't have to rename `linux` into `manylinux` before uploading to PyPI?

Reply:
Correct!

Reply:
Cool, thanks :)

Reply:
The new name is:

> pyflagser-0.3.0-cp37-cp37m-manylinux2010_x86_64

while the old one was:

> pyflagser-0.3.0-cp37-cp37m-linux_x86_64.whl

which use to be rename to:

> pyflagser-0.3.0-cp37-cp37m-manylinux_x86_64.whl

Will the extra `2010` pose any issue?

Reply:
I believe it should always have been `manylinux2010` in fact. It has been so in `giotto-tda` for instance, since v0.1.4.

Reply:
Ok, `pyflagser` was working fine on linux anyways it seems.

Reply:
Great, thanks for checking. Ok to merge?

Reply:
Oh, great :)